### ISSUE TITLE
Audio streaming doesn't work. [Vitamio 5.0]

### ISSUE THREAD

**Issue by iosifnicolae2** [VitamioCustom/Vitamio-5.0.0#3]
```
12-27 20:04:26.865 21057-21057/ro.package_name E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ffmpeg, ff_ass_split
12-27 20:04:26.865 21057-21057/ro.package_name E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ffmpeg, ff_ass_split_dialog
12-27 20:04:26.865 21057-21057/ro.package_name E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ffmpeg, ff_ass_split_free
12-27 20:04:26.865 21057-21057/ro.package_name E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ffmpeg, av_close_input_file
12-27 20:04:26.866 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: LOAD FFMPEG END: /data/user/0/ro.package_name/lib/libffmpeg.so
12-27 20:04:26.866 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: LOAD VVO START: /data/user/0/ro.package_name/lib/libvvo.9.so
12-27 20:04:26.873 21057-21057/ro.package_name E/Vitamio[5.0.0][Player]: FIND_NAME_SYM vvo, render_yuv
12-27 20:04:26.873 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: LOAD VVO END: /data/user/0/ro.package_name/lib/libvvo.9.so
12-27 20:04:26.873 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: LOAD VAO START: /data/user/0/ro.package_name/lib/libvao.0.so
12-27 20:04:26.876 21057-25200/ro.package_name I/ProviderInstaller: Installed default security provider GmsCore_OpenSSL
12-27 20:04:26.879 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: LOAD VAO END: /data/user/0/ro.package_name/lib/libvao.0.so
12-27 20:04:26.880 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: VPLAYER INIT BEGIN
12-27 20:04:26.880 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: Vitamio Version 5.0.0 
12-27 20:04:26.880 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: Application package name: ro.package_name
12-27 20:04:26.880 21057-21057/ro.package_name D/Vitamio[5.0.0][Player]: RENDER RGB
12-27 20:04:26.880 21057-25208/ro.package_name D/Vitamio[5.0.0][Player]: THREAD NOTIFY START
12-27 20:04:26.886 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: VPLAYER INIT END
12-27 20:04:26.886 21057-25208/ro.package_name D/Vitamio[Player]: CPU implementer:0x51
12-27 20:04:26.886 21057-25208/ro.package_name D/Vitamio[Player]: CPU architecture:7
12-27 20:04:26.886 21057-25208/ro.package_name D/Vitamio[Player]: processor:3
12-27 20:04:26.886 21057-25208/ro.package_name D/Vitamio[Player]: BogoMIPS:38.40
12-27 20:04:26.886 21057-25208/ro.package_name D/Vitamio[Player]: Revision:0000
12-27 20:04:26.886 21057-25208/ro.package_name D/Vitamio[Player]: CPU revision:1
12-27 20:04:26.886 21057-25208/ro.package_name D/Vitamio[Player]: Serial:0000000000000000
12-27 20:04:26.886 21057-25208/ro.package_name D/Vitamio[Player]: Hardware:Qualcomm MSM8974PRO-AC
12-27 20:04:26.886 21057-25208/ro.package_name D/Vitamio[Player]: CPU variant:0x2
12-27 20:04:26.886 21057-25208/ro.package_name D/Vitamio[Player]: Processor:ARMv7 Processor rev 1 (v7l)
12-27 20:04:26.886 21057-25208/ro.package_name D/Vitamio[Player]: Features:swp half thumb fastmult vfp edsp neon vfpv3 tls vfpv4 idiva idivt vfpd32
12-27 20:04:26.886 21057-25208/ro.package_name D/Vitamio[Player]: CPU part:0x06f
12-27 20:04:26.887 21057-25208/ro.package_name D/Vitamio[Player]: CPU architecture: 7
12-27 20:04:26.887 21057-25208/ro.package_name D/Vitamio[Player]: GET CPU FATURE: V5TE V6 VFP V7A VFPV3 NEON 
12-27 20:04:27.563 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]:  stop player, stop to play 
12-27 20:04:27.563 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: stop
12-27 20:04:27.563 21057-21057/ro.package_name E/Vitamio[5.0.0][Player]: stop called in state 1
12-27 20:04:27.565 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: Copyright (c) YIXIA (http://yixia.com).
                                                                                      THIS SOFTWARE (Vitamio) IS WORK OF YIXIA (http://yixia.com)
12-27 20:04:27.565 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: Vitamio Version 5.0.0 
12-27 20:04:27.565 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: Application package name: ro.package_name
12-27 20:04:27.565 21057-21057/ro.package_name E/Vitamio[5.0.0][Player]: SURFACE INIT FAILED
12-27 20:04:27.565 21057-21057/ro.package_name D/Vitamio[5.0.0][Player]: prepare
12-27 20:04:27.565 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: PREPARE SIG: 0
12-27 20:04:27.566 21057-21057/ro.package_name I/Vitamio[5.0.0][Player]: start
12-27 20:04:27.566 21057-21057/ro.package_name E/Vitamio[5.0.0][Player]: start called in state 4
12-27 20:04:27.567 21057-25228/ro.package_name I/Vitamio[5.0.0][Player]: THREAD PREPARE START
12-27 20:04:27.567 21057-25228/ro.package_name I/Vitamio[5.0.0][Player]: THREAD PREPARE ATTACHED
12-27 20:04:27.568 21057-25228/ro.package_name I/Vitamio[5.0.0][Player]: Copyright (c) YIXIA (http://yixia.com).
                                                                                      THIS SOFTWARE (Vitamio) IS WORK OF YIXIA (http://yixia.com)
12-27 20:04:27.568 21057-25228/ro.package_name I/Vitamio[5.0.0][Player]: Vitamio Version 5.0.0 
12-27 20:04:27.568 21057-25228/ro.package_name I/Vitamio[5.0.0][Player]: OPEN FILE http://mobile.url_stream_here.ro:8127
12-27 20:04:27.629 21057-25228/ro.package_name E/Vitamio[5.0.0][Player]: [aac @ 0xb37ffc00] channel element 1.14 is not allocated
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: Input #0, aac, from 'http://mobile.url_stream_here.ro:8127':
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]:   Metadata:
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]:     icy-notice1     : 
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: <BR>This stream requires <a href="http://www.winamp.com/">Winamp</a><BR>
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]:     icy-notice2     : 
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: SHOUTcast Distributed Network Audio Server/Linux v1.9.8<BR>
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]:     icy-name        : 
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: Station name
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]:     icy-genre       : 
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: christian
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]:     icy-url         : 
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: http://www.url_stream_here.ro
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]:     icy-pub         : 
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: 1
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]:     icy-br          : 
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: 64
12-27 20:04:27.646 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]:     StreamTitle     : 
12-27 20:04:27.647 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: Nelu Andresoi - Track 10 [ www.url_stream_here.ro ]
12-27 20:04:27.647 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]:     StreamUrl       : 
12-27 20:04:27.647 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: http://www.url_stream_here.ro
12-27 20:04:27.647 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]:   Duration: 
12-27 20:04:27.647 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: N/A
12-27 20:04:27.647 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: , bitrate: 
12-27 20:04:27.647 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: 52 kb/s
12-27 20:04:27.647 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]:     Stream #0:0
12-27 20:04:27.647 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: : Audio: aac (HE-AAC), 44100 Hz, stereo, fltp, 52 kb/s
12-27 20:04:27.647 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: METADATA: <BR>This stream requires <a href="http://www.winamp.com/">Winamp</a><BR>SHOUTcast Distributed Network Audio Server/Linux v1.9.8<BR> Mobilechristianhttp://www.url_stream_here.ro164[ www.url_stream_here.ro ]http://www.url_stream_here.ro
12-27 20:04:27.647 21057-25228/ro.package_name I/Vitamio[5.0.0][Player]: META AUDIO: 0. und, META SUBTITLE: 
12-27 20:04:27.647 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: FIND CODEC_ID 86018
12-27 20:04:27.647 21057-25228/ro.package_name I/Vitamio[5.0.0][Player]: WOW: BRILLIANT C COUNT 4
12-27 20:04:27.649 21057-25228/ro.package_name D/Vitamio[5.0.0][Player]: OPEN CODEC aac
12-27 20:04:27.649 21057-25228/ro.package_name I/Vitamio[5.0.0][Player]: OPEN STREAM AUDIO BEGIN
12-27 20:04:27.650 21057-25233/ro.package_name D/Vitamio[5.0.0][Player]: THREAD AUDIO_DECODE START
12-27 20:04:27.650 21057-25228/ro.package_name I/Vitamio[5.0.0][Player]: Copyright (c) YIXIA (http://yixia.com).
                                                                                      THIS SOFTWARE (Vitamio) IS WORK OF YIXIA (http://yixia.com)
12-27 20:04:27.650 21057-25228/ro.package_name I/Vitamio[5.0.0][Player]: Vitamio Version 5.0.0 
12-27 20:04:27.650 21057-25228/ro.package_name I/Vitamio[5.0.0][Player]: Application package name: ro.package_name
12-27 20:04:27.650 21057-25228/ro.package_name I/Vitamio[5.0.0][Player]: 2, 48000, 0
12-27 20:04:27.659 21057-21057/ro.package_name I/Vitamio[Player]: Info (704, 0)
12-27 20:04:27.660 21057-21057/ro.package_name W/info listener: 704  0
```

Please resolve this issue.

And one another issue when i comile my android package in Android M says that libffmpeg has text realocations, how to solve this ?

**Comment by dorkden** [comment]
hello! did you solve this error yet?

**Comment by gturedi** [comment]
same erorror

**Comment by dorkden** [comment]
for me I fixed it by using MediaPlayer instead of VideoView

**Comment by abbas999** [comment]
Any ideas about this?? I'm getting the errors below and no videos are playing:

```
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ffmpeg, ff_ass_split
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ffmpeg, ff_ass_split_dialog
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ffmpeg, ff_ass_split_free
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ffmpeg, av_close_input_file
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, SSL_shutdown
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, SSL_free
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, SSL_CTX_free
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, SSL_load_error_strings
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, SSL_library_init
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, SSL_CTX_new
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, SSLv23_client_method
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, SSL_new
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, SSL_set_fd
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, SSL_connect
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, SSL_read
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, SSL_write
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM ssl, ERR_print_errors_fp
02-19 14:15:18.760 3423-3423/? E/Vitamio[5.0.0][Player]: FIND_NAME_SYM vvo, render_yuv
02-19 14:15:18.780 3423-3521/? E/Vitamio[5.0.0][Player]: avformat_open_input: No such file or directory : -2
02-19 14:15:18.790 3423-3520/? E/Vitamio[5.0.0][Player]: error (1, -2)
02-19 14:15:20.840 3423-3423/? E/Vitamio[Player]: Error (1, -2)
```

**Comment by kadriengin** [comment]
For vitamio 5.0 audio streams you must add this function to your player code after  "mMediaPlayer.setOnVideoSizeChangedListener(this);"  this

Add this code and vitamio 5.0 vill play streams with audio...
            mMediaPlayer.setOnInfoListener(new MediaPlayer.OnInfoListener() {
                @Override
                public boolean onInfo(MediaPlayer mp, int what, int extra) {
                    switch (what) {
                        case MediaPlayer.MEDIA_INFO_FILE_OPEN_OK: // line added 1
                            Log.d("XXX", ">>>>>>>>>>>>>>>>>>>>>>>> onInfo: MEDIA_INFO_FILE_OPEN_OK");
                            long buffersize = mMediaPlayer.audioTrackInit(); // line added 2
                            mMediaPlayer.audioInitedOk(buffersize); // line added 3
                            break;
                    }
                    return true;
                }
            });

**Comment by MythodeaLoL** [comment]
See [Vitamio 5.0 Audio problem with video streaming. How can I fix it? **#5**](https://github.com/VitamioCustom/Vitamio-5.0.0/issues/5).
